cmd/ttn-lw-cli/commands/flags.go

```diff
@@ -129,6 +130,19 @@ func getAPIKeyID(flagSet *pflag.FlagSet, args []string, i int) string {
 	return apiKeyID
 }
 
+func getPlainTextValue(flagSet *pflag.FlagSet, args []string, i int) string {
+	var plainTextValue string
+	if len(args) > 0+i {
+		if len(args) > 1+i {
+			logger.Warn("Multiple plain text values found in arguments, considering only the first")
+		}
+		plainTextValue = args[0+i]
+	} else {
+		plainTextValue, _ = flagSet.GetString("plain-text-value")
```

Comment:
It's a bad practice to take secrets from flags. Can we avoid this by piping or prompt? Or would that become really unfriendly?

Reply:
Yeah indeed. I forgot to remove it. We get the secret value using `gopass` similar to the passwords.